packages/shared/describeComponentFrame.js

```diff
@@ -17,7 +17,7 @@ export default function(
     (name || 'Unknown') +
     (source
       ? ' (at ' +
-        source.fileName.replace(/^.*[\\\/]/, '') +
+        source.fileName.match(/[^\\\/]*([\\\/]index\.[^\\\/]+)?$/)[0] +
```

Comment:
Could you add a comment with different things it’s supposed to match?

Reply:
Done :)